core/src/main/resources/hudson/model/CauseAction/summary.jelly

```diff
@@ -24,14 +24,32 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <t:summary icon="orange-square.png">
-    <j:forEach var="entry" items="${it.causeCounts.entrySet()}">
-      <p>
-        <st:include page="description.jelly" it="${entry.key}"/>
-        <j:if test="${entry.value > 1}">
-          <st:nbsp/>${%Ntimes(entry.value)}
-        </j:if>
-      </p>
-    </j:forEach>
-  </t:summary>
+	<t:summary icon="orange-square.png">
+	<h4>${%Cause}</h4>
+	<style type="text/css">
+	div.tab:not(:target) div.content, div.tab:target div.show, div.tab div.hide {
+		display: none;
+	}
+	div.tab:target div.content,  div.tab:target div.hide {
+		display: block;
+	}
+	</style>
+	<div class="tab" id="text1">
+		<div class="show">		
+			<a href="#text1">${%Show}</a>
+			<br> </br>
+			</div>
+		<div class="hide">
+			<a href="#">${%Hide}</a>
+		</div>   
+		<div class="content">
```

Comment:
Yes, that would be great, but there is no way to get the information about upstream builds from the causes list item, which is the only one I have at this point - when the yelly is displayed. While the Build is running I get the information about the upstream builds. but when it's done, maybe some days ago there is no way to get information about the upstream builds from the cause item which is contained in the causes list.
To get this it has to be stored earlier in the build process and hast to be accessible from the cause object. But this would be a much bigger change in more files of jenkins core.